test/e2e/windows/kubelet_stats.go

```diff
@@ -59,7 +59,7 @@ var _ = SIGDescribe("[Feature:Windows] Kubelet-Stats [Serial]", func() {
 
 				ginkgo.By("Getting kubelet stats 5 times and checking average duration")
 				iterations := 5
-				var totalDurationMs int64
+				var totalDurationMs int
```

Comment:
duration.Milliseconds() returns int64 so we should keep this var int64 as well.
https://pkg.go.dev/time#Duration.Milliseconds

Reply:
This should fix a lot of the build / typecheck errors too